src/gui/editors/AutomationEditor.cpp

```diff
@@ -615,6 +615,91 @@ void AutomationEditor::mousePressEvent( QMouseEvent* mouseEvent )
 
 
 
+void AutomationEditor::mouseDoubleClickEvent(QMouseEvent * mouseEvent)
+{
+	if (!validPattern()) { return; }
+
+	// Helper lambda function to fine tune a node's in/out value
+	auto fineTuneValue = [this](timeMap::iterator node, bool editingInValue)
```

Comment:
Can you extract this to a regular function? It could make sense to call outside of double click events (like via a context menu, keyboard shortcut, or new edit mode).

Reply:
Sure! I'll move it to `AutomationEditor::fineTuneValue` then